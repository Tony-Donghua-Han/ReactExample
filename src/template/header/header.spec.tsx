import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Header from "./header";

describe("Header", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<Header />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
