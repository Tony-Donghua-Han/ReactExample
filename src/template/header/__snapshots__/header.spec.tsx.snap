// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render without crushing 1`] = `
"<div id=\\"header\\" className=\\"header\\">
  <div className=\\"text\\">
    Web Title
  </div>
</div>"
`;
